README.md

```diff
@@ -62,14 +62,15 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - ["Mastering Markdown"](https://guides.github.com/features/mastering-markdown/) by [@GitHubGuides](https://guides.github.com/)
 
 ## Using version control
+- [Try Git](https://try.github.io/) - Learn Git in 15 minutes from within your browser for free.
+- [Everyday Git](https://git-scm.com/docs/giteveryday) - A useful minimum set of commands for Everyday Git
 - [Oh shit, git!](http://ohshitgit.com/) - how to get out of common `git` mistakes described in plain English.
 - [Atlassian Git Tutorials](https://www.atlassian.com/git/tutorials/) - various tutorials on using `git`
 - [GitHub Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) (PDF)
 - [freeCodeCamp's Wiki on Git Resources](https://forum.freecodecamp.org/t/wiki-git-resources/13136)
 - [GitHub Flow](https://www.youtube.com/watch?v=juLIxo42A_s) (42:06) - GitHub talk on how to make a pull request.
 - [GitHub Learning Resources](https://help.github.com/articles/git-and-github-learning-resources/) - Git and GitHub learning resources.
 - [Pro Git](https://git-scm.com/book/en/v2) - The entire Pro Git book, written by Scott Chacon and Ben Straub and published by Apress.
-- [Try Git](https://try.github.io/) - Learn Git in 15 minutes from within your browser for free.
```

Comment:
Moving this as 1st item in the section as seems very useful for introduction. Please let me know if this change should not be included with this commit.